code/modules/mob/living/simple_animal/hostile/beheaded_kamikaze.dm

```diff
@@ -0,0 +1,30 @@
+/mob/living/simple_animal/hostile/beheaded_kamikaze
+	name = "beheaded kamikaze"
+	desc = "You can't be serious."
+	icon_state = "clown" //placeholder
+	icon_living = "clown" //placeholder
+	speak_chance = 80
+	speak = list("AAAAAAAAAAAAAAAAA", "AAAAAAAAAAAAAAAAAAAAAA", "AAAAAAAA", "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA", "AAAAAAAAAA")
```

Comment:
Can I suggest replacing these with something a little more fluffy? And maybe use a loop to go:

```
var/taunt_length = rand(10, 20)
var/i
var/taunt = "AAAAA"

for(var/i = 5; i < taunt_length; i++)
    taunt += 'A'

taunt += 'GH!!!'
```

So you end up with differing lengths of "AAAAAAAAGH!!!"